guides-markdown/segwit.md

```diff
@@ -356,29 +381,14 @@ Redeem script for P2WSH is in the witness with its signature.
 
 ### P2SH Nested
 
-Bcoin MTX and KeyRing primitives construct all necessary scripts for us, so the only thing
-that changes when moving to nested segwit addresses is the UTXO script.
+Bcoin MTX and KeyRing primitives construct all necessary scripts for us, so the thing
+that changes when moving to nested segwit addresses is the UTXO pubkeyScript.
 
-So instead of using `ring.getAddress()`...
-```js
-const address = ring.getAddress();
-
-console.log(`Address where we received money: ${address}`);
+To spend P2SH inputs using bcoin change is small, we'll need
```

Comment:
change to -> "To spend P2SH inputs using bcoin, we just need to use `ring.getNestedAddress` instead of `ring.getAddress()` or set the `ring.nested` variable to `true` and bcoin can handle it automatically with `ring.getAddress()`.